apps/twoopstracker/contrib/dokku/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM codeforafrica/twoopstracker-ui:0.1.0
```

Comment:
We also no longer need this file since we dokku/github-action supports pushing Docker images directly